### PR TITLE
Add The First osu! Manga: osu! Combat Championship news post

### PR DESCRIPTION
**Pull request by Vortex100K**
Add The First osu! Manga: osu! Combat Championship news post

needs to add poster images

### CODE REVIEW

**Review by Blooshing — CHANGES_REQUESTED** [review]

File: wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/umbreEcard.png
Comment (Hiviexd) [inline wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/umbreEcard.png:1]: this has `mrekk` as the background text, doesn't seem intentional

(it also might be missing the top background text that's present in the mrekk and kariyu banners?)

Comment (Vortex100K) [reply wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/umbreEcard.png:1]: oh yikes ill get to fixing that right away lol

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+title: "The First osu! Manga: osu! Combat Championship"
+date: 2023-10-21 13:00:00 +0000
+---
+
+WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted.
+
+![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/OCCnewspostrecent.jpg)
+
+**Calling all osu! enthusiasts!**
+
```
Comment (0x84f) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:12]:
```suggestion

```
was a space in there

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  </a>
+</p>
+
+Curious about which of your favorite players are in The osu! Combat Championship manga? Check out the character introductions below:
+
+[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/whitecatbanner.jpg)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)
+
+WhiteCat: WhiteCat exudes an air of aloofness, often carrying a sense of tension, but beneath the surface lies intelligence and a careful choice of words. His determination and focus were forged through a tragic event in his youth when he lost his brother, BlackDog, in an incident involving mrekk. WhiteCat has harbored blame toward mrekk for his loss, yet he grapples with a moral dilemma: should he seek revenge or aspire to become the greatest osu! player through fair means? At the age of 15, WhiteCat's uncontrollable power led to a berserk episode, with peppy being the only one capable of stopping him.
```
Comment (0x84f) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:29]: I am not sure if the `player:` thing is necessary if you have the description under the banner and it mentions them by name
```suggestion
WhiteCat exudes an air of aloofness, often carrying a sense of tension, but beneath the surface lies intelligence and a careful choice of words. His determination and focus were forged through a tragic event in his youth when he lost his brother, BlackDog, in an incident involving mrekk. WhiteCat has harboured blame toward mrekk for his loss, yet he grapples with a moral dilemma: should he seek revenge or aspire to become the greatest osu! player through fair means? At the age of 15, WhiteCat's uncontrollable power led to a berserk episode, with peppy being the only one capable of stopping him.
```

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/mrekkcard.png)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)
+
+mrekk: A true businessman, willing to go to any lengths to secure profits and success. He seldom finds time or inclination for frivolous pursuits, such as relationships or family, as his primary focus is on strengthening his business. His unwavering desire for profit is rooted in a profound fear of poverty, stemming from a childhood trauma that has left a lasting impact.
```
Comment (0x84f) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:33]:
```suggestion
A true businessman, mrekk is willing to go to any lengths to secure profits and success. He seldom finds time or inclination for frivolous pursuits, such as relationships or family, as his primary focus is on strengthening his business. His unwavering desire for profit is rooted in a profound fear of poverty, stemming from a childhood trauma that has left a lasting impact.
```

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/umbreEcard.png)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)
+
+Umbre: On the surface, he's a bit of a jokester, often cracking light-hearted quips at WhiteCat and creating a generally pleasant atmosphere. He serves as an exceptionally effective mentor, stepping into the role of WhiteCat's brother-figure after the loss of BlackDog and helping to nurture him into the state he's in today. Umbre suffers an internal struggle, torn between being Umbre the player or Umbre the brother. Nonetheless, his kind-hearted nature prevails, leading him to make the decision to guide WhiteCat to heights he could never have imagined.
```
Comment (0x84f) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:37]:
```suggestion
On the surface, Umbre is a bit of a jokester, often cracking light-hearted quips at WhiteCat and creating a generally pleasant atmosphere. He serves as an exceptionally effective mentor, stepping into the role of WhiteCat's brother-figure after the loss of BlackDog and helping to nurture him into the state he's in today. Umbre suffers an internal struggle, torn between being Umbre the player or Umbre the brother. Nonetheless, his kind-hearted nature prevails, leading him to make the decision to guide WhiteCat to heights he could never have imagined.
```

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/kariyunewcard.jpg)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)
+
+Kariyu: An extroverted and playful individual. He revels in being a jokester and easily captivates a broad audience. As a child, he harbored a strong desire to emulate his idol, Vaxei. In pursuit of this dream, he embarked on a solo journey to the new continent and returned with a newfound companion, Cooper. Together, they have embarked on countless adventures and taken part in numerous tournaments.
```
Comment (0x84f) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:41]:
```suggestion
Kariyu is an extroverted and playful individual. He revels in being a jokester and easily captivates a broad audience. As a child, he harboured a strong desire to emulate his idol, Vaxei. In pursuit of this dream, he embarked on a solo journey to the new continent and returned with a newfound companion, Cooper. Together, they have embarked on countless adventures and taken part in numerous tournaments.
```

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/xootycard.jpg)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)
+
+xootynator: An extroverted and composed individual, xootynator exudes boundless optimism. Her quest for enjoyment is inseparable from her determination to make her mark in the predominantly male-dominated arena of combat. Fueled by the excitement of battle and a commitment to shatter gender barriers, xootynator also finds immense joy in the world of art, with a particular fondness for birds.
```
Comment (0x84f) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:45]:
```suggestion
An extroverted and composed individual, xootynator exudes boundless optimism. Her quest for enjoyment is inseparable from her determination to make her mark in the predominantly male-dominated arena of combat. Fuelled by the excitement of battle and a commitment to shatter gender barriers, xootynator also finds immense joy in the world of art, with a particular fondness for birds.
```

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+Meet WhiteCat, mrekk, and a bunch of other interesting characters in the OCC manga! Each one's got their own unique style and story. As you flip through those character panels, you're in for a diverse crew — some cool, some quirky, but all ready to bring their own flavor to the OCC manga!. It's a character buffet of people you already know, so get ready to dig in and enjoy the ride!
```
Comment (0x84f) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:47]:
```suggestion
Meet WhiteCat, mrekk, and a bunch of other interesting characters in the OCC manga! Each one's got their own unique style and story. As you flip through those character panels, you're in for a diverse crew — some cool, some quirky, but all ready to bring their own flavour to the OCC manga! It's a character buffet of people you already know, so get ready to dig in and enjoy the ride!
```

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+<p align="center">
+  <img width="98%" height="98%" src="/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/occteamnew.png">
+</p>
+
+---
+
+If you are a top 50 player and want to be a part of the series, or someone who wants to participate in the production of OCC, feel free to reach out to [VTX](https://osu.ppy.sh/users/17539772) on any platform. The team is open to collaborations and welcomes anyone with experience.
+
+Prepare yourselves for an unforgettable adventure as osu! Combat Championship takes center stage. Stay tuned for further updates on the manga!
```
Comment (0x84f) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:61]:
```suggestion
Prepare yourselves for an unforgettable adventure as osu! Combat Championship takes centre stage. Stay tuned for further updates on the manga!
```

**Review by 0x84f — CHANGES_REQUESTED** [review]
Suggested the following:
- change wording to bri'ish versions
- remove the `Player:` as it feels unnecessary right under their banners and them being explicitly mentioned
- minor cleanup

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted.
+
+![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/OCCnewspostrecent.jpg)
+
+**Calling all osu! enthusiasts!**
+
+Prepare yourselves for a journey like none other, as we introduce the highly anticipated **osu! Combat Championship** (OCC) manga!
+
+ Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, immersing themselves in a once-in-a-lifetime combat-style tournament where participants battle it out for the ultimate goal: attaining the prestigious rank 1 title and having any wish granted. Get ready to experience exhilarating art, intense rivalries, and delve deeper into our unique spin on the world of osu!.
```
Comment (Hiviexd) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:15]:
```suggestion
Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, immersing themselves in a once-in-a-lifetime combat-style tournament where participants battle it out for the ultimate goal: attaining the prestigious rank 1 title and having any wish granted. Get ready to experience exhilarating art, intense rivalries, and delve deeper into our unique spin on the world of osu!.
```

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+<p align="center">
+  <a href="https://mangadex.org/title/f1d50eba-6ace-4490-8439-07692fda3b9c/osu-combat-championship">
+    <img src="/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/occbuttonnew.png" alt="Character Panels">
+  </a>
+</p>
+
+Curious about which of your favorite players are in The osu! Combat Championship manga? Check out the character introductions below:
+
+[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/whitecatbanner.jpg)](https://media.discordapp.net/attachments/1003689172420276246/1159889762602205265/hHq0xR12mgffBIrgk6X53ChtAy0o_dxYDOLNjkf0d8s.png?ex=6532aa85&is=65203585&hm=20b690670640f90a93b811130c3b746d1d10a9824b9da20ab2e3f237a27e69a4&=&width=456&height=675)
```
Comment (Hiviexd) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:27]: not sure what's going on with each banner linking to some kind of temporary photo. it'd be better to make them link to each player's osu! profile if you really want to make them clickable.

**same applies to other banners.**

### without link:
```suggestion
![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/whitecatbanner.jpg)
```
### with link:
```suggestion
[![](/wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/whitecatbanner.jpg)](https://osu.ppy.sh/users/4504101)
```

File: wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/mrekkcard.png
Comment (Hiviexd) [inline wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/mrekkcard.png:1]: any reason why this banner is `png` while other player banners are in `jpg`? would be better to make it `jpg` for consistency and to save up on space

File: wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/umbreEcard.png
Comment (Hiviexd) [inline wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/umbreEcard.png:1]: any reason why this banner is `png` while other player banners are in `jpg`? would be better to make it `jpg` for consistency and to save up on space

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "The First osu! Manga: osu! Combat Championship"
+date: 2023-10-21 13:00:00 +0000
+---
+
+WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted.
```
Comment (Hiviexd) [inline news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:7]:
```suggestion
WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted!
```

**Review by Hiviexd — CHANGES_REQUESTED** [review]

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md (hunk 10 above)
Comment (Walavouchey) [reply news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:27]: these are meant to link to full-size versions of the images

Comment (Vortex100K) [reply news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:27]: Yeah lol

Comment (Vortex100K) [reply news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:27]: > these are meant to link to full-size versions of the images

Yeah lol

File: wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/mrekkcard.png
Comment (Vortex100K) [reply wiki/shared/news/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship/mrekkcard.png:1]: no just a mismatch on my part if its necessary ill change to jpg

File: news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md (hunk 10 above)
Comment (Hiviexd) [reply news/2023/2023-10-21-The-First-osu!-Manga-osu!-Combat-Championship.md:27]: will keep this unresolved until the temp images are replaced with open ones then

**Review by Hiviexd — APPROVED** [review]

**Comment by Hiviexd** [comment]
currently waiting on a final wala check btw, thought i'd communicate that

File: news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+date: 2023-11-17 13:00:00 +0000
+---
+
+WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted!
+
+![](/wiki/shared/news/2023-11-17-the-first-osu-manga-osu-combat-championship/banner.jpg)
+
+**Calling all osu! enthusiasts!**
+
+Prepare yourselves for a journey like none other, as we introduce the highly anticipated **osu! Combat Championship** (OCC) manga!
```
Comment (Walavouchey) [inline news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md:13]:
```suggestion
Prepare yourselves for a journey like none other, as we introduce **osu! Combat Championship** (OCC)!
```

File: news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "The First osu! Manga: osu! Combat Championship"
+date: 2023-11-17 13:00:00 +0000
+---
+
+WhiteCat immerses himself in a once-in-a-lifetime combat-style tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted!
```
Comment (Walavouchey) [inline news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md:7]:
```suggestion
WhiteCat immerses himself in a once-in-a-lifetime combat tournament. The stakes are high as he battles it out to claim the coveted #1 title and have his wish granted!
```

File: news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+
+![](/wiki/shared/news/2023-11-17-the-first-osu-manga-osu-combat-championship/banner.jpg)
+
+**Calling all osu! enthusiasts!**
+
+Prepare yourselves for a journey like none other, as we introduce the highly anticipated **osu! Combat Championship** (OCC) manga!
+
+Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, immersing themselves in a once-in-a-lifetime combat-style tournament where participants battle it out for the ultimate goal: attaining the prestigious rank 1 title and having any wish granted. Get ready to experience exhilarating art, intense rivalries, and delve deeper into our unique spin on the world of osu!.
```
Comment (Walavouchey) [inline news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md:15]: repetition

```suggestion
Join us as we delve into the captivating storyline that follows the path of WhiteCat, a skilled player, taking on formidable enemies in a tournament for the ultimate prize: attaining the prestigious rank #1 title and having any wish granted. Get ready to experience exhilarating art, intense rivalries, and delve deeper into a unique spin on the world of osu!.
```

File: news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+
+For those eager to dive into the manga and skip the introductions, simply click the button below to head over to Mangadex and start reading.
+
+<p class="osu-md__paragraph" align="center">
+  <a href="https://mangadex.org/title/f1d50eba-6ace-4490-8439-07692fda3b9c/osu-combat-championship">
+    <img src="/wiki/shared/news/2023-11-17-the-first-osu-manga-osu-combat-championship/read-now.png" alt="Click here to read">
+  </a>
+</p>
+
+Curious about which of your favorite players are in The osu! Combat Championship manga? Check out the character introductions below:
```
Comment (Walavouchey) [inline news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md:25]:
```suggestion
Curious about which of your favourite players will partake in the osu! Combat Championship? Read on below to find out and click the banners to view the full artworks.
```

File: news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+
+[![](/wiki/shared/news/2023-11-17-the-first-osu-manga-osu-combat-championship/WhiteCat.jpg)](https://assets.ppy.sh/media/news/manga-occ/WhiteCat.png)
+
+WhiteCat exudes an air of aloofness, often carrying a sense of tension, but beneath the surface lies intelligence and a careful choice of words. His determination and focus were forged through a tragic event in his youth when he lost his brother, BlackDog, in an incident involving mrekk. WhiteCat has harboured blame toward mrekk for his loss, yet he grapples with a moral dilemma: should he seek revenge or aspire to become the greatest osu! player through fair means? At the age of 15, WhiteCat's uncontrollable power led to a berserk episode, with Peppy being the only one capable of stopping him.
```
Comment (Walavouchey) [inline news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md:29]:
```suggestion
WhiteCat exudes an air of aloofness, often carrying a sense of tension, but beneath the surface lies intelligence and a careful choice of words. His determination and focus were forged through a tragic event in his youth when he lost his brother, BlackDog, in an incident involving mrekk. WhiteCat has harboured blame toward mrekk for his loss, yet he grapples with a moral dilemma: should he seek revenge, or aspire to become the greatest osu! player through fair means? At the age of 15, WhiteCat's uncontrollable power led to a berserk episode, with peppy being the only one capable of stopping him.
```

File: news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+
+[![](/wiki/shared/news/2023-11-17-the-first-osu-manga-osu-combat-championship/Kariyu.jpg)](https://assets.ppy.sh/media/news/manga-occ/Kariyu.png)
+
+Kariyu is an extroverted and playful individual. He revels in being a jokester and easily captivates a broad audience. As a child, he harboured a strong desire to emulate his idol, Vaxei. In pursuit of this dream, he embarked on a solo journey to the new continent and returned with a newfound companion, Cooper. Together, they have embarked on countless adventures and taken part in numerous tournaments.
+
+[![](/wiki/shared/news/2023-11-17-the-first-osu-manga-osu-combat-championship/xootynator.jpg)](https://assets.ppy.sh/media/news/manga-occ/xootynator.png)
+
+An extroverted and composed individual, xootynator exudes boundless optimism. Her quest for enjoyment is inseparable from her determination to make her mark in the predominantly male-dominated arena of combat. Fuelled by the excitement of battle and a commitment to shatter gender barriers, xootynator also finds immense joy in the world of art, with a particular fondness for birds.
+
```
Comment (Walavouchey) [inline news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md:46]:
```suggestion

---

```

File: news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+Meet WhiteCat, mrekk, and a bunch of other interesting characters in the OCC manga! Each one's got their own unique style and story. As you flip through those character panels, you're in for a diverse crew — some cool, some quirky, but all ready to bring their own flavour to the OCC manga! It's a character buffet of people you already know, so get ready to dig in and enjoy the ride!
```
Comment (Walavouchey) [inline news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md:47]:
```suggestion
Meet WhiteCat, mrekk, and a bunch of other interesting characters in the OCC manga! Each one's got their own unique style and story. If you didn't already notice from the character introductions, you're in for a diverse crew — some cool, some quirky, but all ready to bring their own flavour to the story! It's a character buffet of people you already know, so get ready to dig in and enjoy the ride!
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md (hunk 16 above)
Comment (Vortex100K) [reply news/2023/2023-11-17-the-first-osu-manga-osu-combat-championship.md]: peppy

**Review by Walavouchey — APPROVED** [review]

**Review by Ephemeralis — DISMISSED** [review]
the OCC team listing image looks a *hair* small but i struggle to think as to how it can realistically be addressed, so this is fine

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**